src/stores/FormState/types.ts

```diff
@@ -50,6 +50,12 @@ export interface EntityFormState {
     isIdle: boolean;
     isActive: boolean;
 
+    showPreSavePrompt: boolean;
+    setShowPreSavePrompt: (data: EntityFormState['showPreSavePrompt']) => void;
+
+    liveSpec: Schema | null;
+    setLiveSpec: (data: EntityFormState['liveSpec']) => void;
```

Comment:
Need a place to store this and I honestly don't care where cause eventually all the create/edit stores need to end up being in one big store anyway so added it here.